packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -470,6 +470,20 @@ export interface CallWithChatAdapterManagement {
    * @beta
    */
   submitSurvey(survey: CallSurvey): Promise<CallSurveyResponse | undefined>;
+  /* @conditional-compile-remove(spotlight) */
+  /**
+   * Start spotlight
+   *
+   * @beta
+   */
+  startSpotlight(userId: string): Promise<void>;
```

Comment:
We've been burned before between using `string` for userId and not something like `CommunicationUserIdentifier` -- should this definitely be string ?

Reply:
(my gut reaction is that is should be some sort of azure communication UserIdentifier type)

Reply:
I agree with that, I think our adapter functions in the past have been using the strings so I guess this impacts whether we would be conforming to what we have done or do we want to change the way we do it in the future and deprecate our old versions taking strings

Reply:
It does not have to be a string. I was debating whether to go with `string` and `CommunicationUserIdentifier` here. I chose `string` to conform to what we have done as Don pointed out. I will think about this some more towards the internal devEx review. So this could be changed in the future.